subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/configuration/Experiments.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.instrumentation.configuration
+
+import java.io.Serializable
+
+public data class Experiments(
```

Comment:
Alternative is to consider it as feature toggles.
"experimental" is mostly about one dimension - longevity.

![](https://martinfowler.com/articles/feature-toggles/chart-4.png)

Reply:
I'm thinking about more refactorings, like use Properties in extensions instead of var's;
Could merge this change while doing it

Reply:
"refactorings" - is also about usage. On this level, I'd like to know nothing about it. It doesn't matter why someone added a flag. The additional knowledge about purpose will be meaningful if it will affect behavior in any way.

Reply:
Added as a "leftover" for this PR